client/src/ui/molecules/a11y-nav/index.tsx

```diff
@@ -24,6 +25,12 @@ export function A11yNav() {
     });
   }
 
+  React.useEffect(() => {
+    if (document && document.location.pathname.includes("docs")) {
```

Comment:
I think we should use `useParams` instead, as that value is reactive, i.e. updates when the site changes, thus triggering that effect to re-run. Right now it only re-runs incidentally.

(another case where we use it, for a different thing https://github.com/mdn/yari/blob/6f66eb8f670699766ea8be46a50accf72ddafe1d/client/src/document/toolbar/edit-actions.tsx#L57)

Reply:
As discussed on Slack I switched to using `useLocation`